src/pages/jam/components/PianoKey.svelte

```diff
@@ -0,0 +1,80 @@
+<script lang="ts">
+    import type { PianoKeyNote } from 'src/models/jam';
+    import { JamPianoStore } from '../../../store/jam';
+
+    export let key: PianoKeyNote;
+
+    function handleKeydown() {
+        JamPianoStore.set({ keydown: true, currNote: key });
+    }
+
+    function handleKeyOver() {
+        if ($JamPianoStore.keydown) {
+            JamPianoStore.update((v) => {
+                return {
+                    ...v,
+                    currNote: key,
+                };
+            });
+        }
+    }
```

Comment:
I thnk you can invert the condition, early return, and get rid of a level of nesting
```suggestion
    function handleKeyOver() {
        if (!$JamPianoStore.keydown) {
            return 
        }
        JamPianoStore.update((v) => {
            return {
                ...v,
                currNote: key,
            };
        });
    }
}
```